### ISSUE TITLE
Open endpoint to easily regrade submissions

### ISSUE THREAD

**Issue by geshuming** [source-academy/cadet#502]
Currently there is no endpoint. If we want to regrade all submissions belong to a mission we need to use the following call:

```elixir
Cadet.Assessments.Submission
|> where(assessment_id: <id>)
|> Repo.all()
|> Enum.each(&Cadet.Autograder.GradingJob.force_grade_individual_submission/1)
```

**Comment by jiachen247** [comment]
maybe its really time for an admin portal haha so everyone doesn't have to go through jiachen :-)

**Comment by martin-henz** [comment]
Yes, let's think of this in our module post-mortem. We need to gather requirements for an admin portal.

**Comment by jiachen247** [comment]
:+1:

**Comment by jiachen247** [comment]
Cadet.Assessments.Submission \
|> where(assessment_id: 392) \
|> Repo.all() \
|> Enum.each(&Cadet.Autograder.GradingJob.force_grade_individual_submission/1)

**Comment by angelsl** [comment]
This is partially done now, just missing an endpoint to regrade all submissions of an assessment.
